libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logger/SFLogger.m

```diff
@@ -65,84 +103,106 @@ -(SFLogLevel)logLevel {
 -(void)setLogLevel:(SFLogLevel) logLevel {
     [self.logger setLogLevel:logLevel];
 }
+
+- (void)submitLogEntryWithClass:(nonnull Class)cls level:(SFLogLevel)level message:(nonnull NSString *)message {
```

Comment:
@bbirman - Here are two new private submit log entry methods that help increase our code reuse.  How do they look?  We do need at least two since they have to fork to the two different signatures in the logger member.  There _are_ ways we could have a single method, but I though the parameter or runtime handling looked less intuitive when I tried it for fun.